Sources/Public/AnimationCache/AnimationCacheProvider.swift

```diff
@@ -11,7 +11,7 @@ import Foundation
 /// can increase performance when loading an animation multiple times.
 ///
 /// Lottie comes with a prebuilt LRU Animation Cache.
-public protocol AnimationCacheProvider {
+public protocol AnimationCacheProvider: AnyObject {
```

Comment:
this is potentially a breaking change, but:

1. I don't think many people are rolling out their own cache providers
2. we kind of need a cache to be a reference type in order to avoid copying all over the place